.github/workflows/build.yml

```diff
@@ -46,6 +46,10 @@ jobs:
         name: Build binaries
         run: |
           make release
+      -
+        name: Build packages
+        run: |
+          make pkg
```

Comment:
Packages will not be included in GitHub Releases (only build). Instead we want to use the remote bake definition from docker-ce-packaging repo.